test/smoke/insight.js

```diff
@@ -1,4 +1,30 @@
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
+const fetch = require('node-fetch');
+
 describe('Insight', () => {
-  // /insight-api-dash/blocks?limit=1
-  it('should return block');
+  networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
+    describe(nodeName, () => {
+      it(`insight should return block ${nodeName}`,
```

Comment:
I think `nodeName` in `describe` is enough?

Reply:
removed